darshan-util/pydarshan/.codecov.yml

```diff
@@ -0,0 +1,22 @@
+# Codecov configuration to make it a bit less noisy
+codecov:
+  # don't fail the CI
+  require_ci_to_pass: yes
```

Comment:
can you explain the reasoning for choosing this setting vs: https://github.com/scipy/scipy/blob/master/codecov.yml#L2

Also, maybe discuss some of the reasons for not adopting the battle-tested settings there or in other OSS projects?